### PR TITLE
Update package.json

### LINKED ISSUE #2932: 🐛 Bug Report: unwanted react-query
### What happened?

In the commit https://github.com/amplication/amplication/commit/6fe3e9522fe9306b3c10f0188f0683f7807cef5f we accidentally added the react-query package to the package.json

### What you expected to happen

not relevant

### How to reproduce

not relevant

### Amplication version

0.13.1

### Environment

_No response_

### Are you willing to submit PR?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by astitva0011**
Close #2932

## PR Checklist
- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

**Review by tupe12334 — CHANGES_REQUESTED** [review]
Please update package-lock file as well

**Review by astitva0011 — COMMENTED** [review]
requested change

**Comment by abrl91** [comment]
@astitva0011 I can't see a change in `package-lock.json`

**Comment by abrl91** [comment]
@astitva0011 I saw you opened a new PR for that. Can you please close the following PR https://github.com/amplication/amplication/pull/4180 and make the change on this PR

**Comment by astitva0011** [comment]
> @astitva0011 I saw you opened a new PR for that. Can you please close the following PR #4180 and make the change oni ha

> @astitva0011 I saw you opened a new PR for that. Can you please close the following PR #4180 and make the change on this PR

i have closed the pr

**Comment by abrl91** [comment]
> > @astitva0011 I saw you opened a new PR for that. Can you please close the following PR #4180 and make the change oni ha
> 
> > @astitva0011 I saw you opened a new PR for that. Can you please close the following PR #4180 and make the change on this PR
> 
> i have closed the pr

Thank you, now you just need to apply the change to the package-lock.json here, on this PR

File: package.json [hunk 1]
```diff
@@ -22,13 +22,6 @@
     "setup:dev": "ts-node ./script/setup.ts",
     "docker:dev": "docker-compose -f docker-compose.dev.yml up -d"
   },
-  "dependencies": {
```
Comment (abrl91) [inline package.json:25]: Why did you remove all the dependencies section?

**Comment by abrl91** [comment]
@astitva0011  on this commit you have the right solution. Just run `npm install` in the root Amplication folder to update the `package-lock.json`

**Pull request closed without merging**